cmd/http/headers.go

```diff
@@ -156,6 +156,9 @@ const (
 	// Reports number of drives currently healing
 	MinIOHealingDrives = "x-minio-healing-drives"
 
+	// Object was stored with additional erasure codes due to degraded system at upload time
+	MinIOErasureUpgraded = "x-minio-erasure-upgraded"
```

Comment:
```suggestion
	MinIOErasureUpgraded = "x-minio-internal-erasure-upgraded"
```

Perhaps also not the right file to keep this since this not a HTTP header